.travis.yml

```diff
@@ -1,19 +1,21 @@
+sudo: false
+os:
+  - linux
```

Comment:
Good question, `linux` is os by default, but I can't find this in the travis docs. Should I remove this line?